README.md

```diff
@@ -16,7 +16,7 @@ functionality. Please use the individual crates listed below instead!
 
 ## Installation
 
-Twilight supports a MSRV of Rust 1.64+.
+Twilight supports a MSRV of Rust 1.67.
```

Comment:
Possible typo? Should be `1.67+`

Reply:
Deliberate. The *minimum required* version entails newer versions are supported, the "+" is therefore superfluous